core-feature-pack/src/main/resources/content/bin/add-user.sh

```diff
@@ -1,12 +1,15 @@
 #!/bin/sh
 
+. ./common.sh
```

Comment:
This needs to happen after the `$DIRNAME` is configured and should be `. "$DIRNAME/common.sh`. Without that it can't be launched from any other directory than bin, e.g. `cd $JBOSS_HOME/bin && ./add-user.sh`.